### PR TITLE
Fix get_coordinate_from_distance() for Arc

### PR DESCRIPTION
**Pull request by JackB-Ansys**
Fixed get_coordinate_from_distance() _BaseArc class method so that correct point on the arc is always returned.

Fixes a bug raised by customer.

### CODE REVIEW

File: src/ansys/motorcad/core/geometry.py
Comment (jgsdavies) [inline src/ansys/motorcad/core/geometry.py:1125]: Update unit test to cover this

**Review by jgsdavies — CHANGES_REQUESTED** [review]

**Review by matthewAnsys — DISMISSED** [review]

File: src/ansys/motorcad/core/geometry.py
Comment (jgsdavies) [inline src/ansys/motorcad/core/geometry.py:1164]: This is in another PR too? 
https://github.com/ansys/pymotorcad/pull/338
We shouldn't commit this in this PR. Either remove manually or wait until #338 is in main and then merge and resolve conflicts

**Review by jgsdavies — CHANGES_REQUESTED** [review]

File: src/ansys/motorcad/core/geometry.py
Comment (JackB-Ansys) [reply src/ansys/motorcad/core/geometry.py:1164]: Yes I think I found this issue while fixing the other one. I think best to wait until 338 is in main then merge this one

**Review by jgsdavies — APPROVED** [review]

**Pull request merged**
